src/feature/relay/router.c

```diff
@@ -2621,16 +2621,15 @@ check_descriptor_ipaddress_changed(time_t now)
       previous = &my_ri->ipv6_addr;
     }
 
-    if (!find_my_address(get_options(), family, LOG_INFO, &current, &method,
-                         &hostname)) {
-      /* Not found, continue onto next family. Logging is done within the
-       * function called. */
-      continue;
-    }
+    /* Ignore returned value because we want to notice not only an address
+     * change but also if an address is lost (current == UNSPEC). */
+    find_my_address(get_options(), family, LOG_INFO, &current, &method,
```

Comment:
It looks like this can leave `current` unset?  That's probably a bug.  We should at least set `current` to AF_UNSPEC or something above.